examples/seismic/acoustic/acoustic_example.py

```diff
@@ -1,5 +1,8 @@
 import numpy as np
-import pytest
+try:
+    import pytest
+except:
```

Comment:
nitpicking: All these imports should be `except ImportError`

Reply:
I thought we needed exactly the opposite?..e.g. pass

Reply:
???